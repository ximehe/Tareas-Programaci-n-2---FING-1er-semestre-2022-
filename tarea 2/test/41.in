# implementar insertarAlFinal
insertarAlFinal 5 5.0
primeroEnCadena
removerDeCadena 5
cantidadEnCadena
insertarAlInicio 6 6.0
insertarAlFinal 7 7.0
primeroEnCadena
imprimirCadena
removerDeCadena 7
primeroEnCadena
insertarAlFinal 6 16.1
primeroEnCadena
cadenaSiguiente 0
cadenaSiguiente 1
reiniciarEstructuras
insertarAlFinal 5 5.0
insertarAlFinal 5 15.1
insertarAlFinal 5 25.2
insertarAlFinal 5 35.3
primeroEnCadena
cadenaSiguiente 0
cadenaSiguiente 3
Fin