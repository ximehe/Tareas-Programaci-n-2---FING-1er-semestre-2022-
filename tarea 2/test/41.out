1>#  implementar insertarAlFinal.
2>Se insertó (5,5.00) al final de la cadena.
3>El primero es (5,5.00)
4>Se removió la primera aparición de 5 de la cadena.
5>La cantidad es 0.
6>Se insertó (6,6.00) al inicio de la cadena.
7>Se insertó (7,7.00) al final de la cadena.
8>El primero es (6,6.00)
9>(6,6.00)(7,7.00)
10>Se removió la primera aparición de 7 de la cadena.
11>El primero es (6,6.00)
12>Se insertó (6,16.10) al final de la cadena.
13>El primero es (6,6.00)
14>(6,6.00)(6,16.10)
15>(6,16.10)(6,6.00)
16>Estructuras reiniciadas.
17>Se insertó (5,5.00) al final de la cadena.
18>Se insertó (5,15.10) al final de la cadena.
19>Se insertó (5,25.20) al final de la cadena.
20>Se insertó (5,35.30) al final de la cadena.
21>El primero es (5,5.00)
22>(5,5.00)(5,15.10)(5,25.20)(5,35.30)
23>(5,35.30)(5,5.00)(5,15.10)(5,25.20)
24>Fin.
